network-api/networkapi/templates/pages/libraries/rcc/author_detail_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "pages/libraries/base.html" %}
+{% load i18n l10n wagtailcore_tags wagtailimages_tags breadcrumbs %}
+
+{% block library_content %}
+    <div
+        id="info"
+        class="
+            tw-gap-8
+            tw-grid
+            tw-grid-cols-1
+            {% if author_profile.image %}
+                medium:tw-gap-x-24
+                medium:tw-grid-cols-[13rem_1fr]
+                large:tw-grid-cols-[16rem_1fr]
+            {% endif %}
+        "
+    >
+        <div
+            {% if author_profile.image %}
+                class="medium:tw-col-start-2"
+            {% endif %}
+        >
+            {% get_rcc_breadcrumbs include_self=True %}
```

Comment:
The `get_rcc_breadcrumbs` tag renders the breadcrumbs directly, there is no need to cast it to a variable. Since this is a routable page, we need `include_self=True` to get the `Browse Authors /` part of the breadcrumbs.

Reply:
Ok. The inclusion makes sense. For clarity though, I would recommend removing the `get_` from the tag. The `get` makes it sound like it's responsible to data retrieval and not rendering.